qase-testng/src/test/java/io/qase/testng/QaseListenerTest.java

```diff
@@ -115,7 +132,8 @@ public void failedWithTimeBulkTest() {
     @Test
     public void allBulkTests() {
         useBulk(true);
-        runTest(Arrays.asList(Passed.class, PassedWithTime.class, Failed.class, FailedWithTime.class));
+        runTest(Arrays.asList(Passed.class, PassedWithTime.class, Failed.class, FailedWithTime.class,
+                WithSteps.class, WithStepsSuccess.class));
         verify(postRequestedFor(urlPathEqualTo("/v1/result/PRJ/777/bulk"))
                 .withHeader("Token", equalTo("secret-token"))
                 .withHeader("Content-Type", equalTo("application/json; charset=UTF-8"))
```

Comment:
I would suggest writing Json into a separate object and then comparing it with an object, not a huge json